_15thnight/migrations/versions/dd2c1b8e5c2c_custom_categories_mtom.py

```diff
@@ -0,0 +1,86 @@
+"""Custom Categories MtoM
+
+Revision ID: dd2c1b8e5c2c
+Revises: a6fbbe49d0bd
+Create Date: 2016-06-28 21:20:13.874600
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'dd2c1b8e5c2c'
+down_revision = 'a6fbbe49d0bd'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'categories',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('name', sa.String(length=32), nullable=False, unique=True),
+        sa.Column('description', sa.Text(), nullable=True),
+        sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table(
+        'user_categories',
+        sa.Column('user_id', sa.Integer(), nullable=True),
+        sa.Column('category_id', sa.Integer(), nullable=True),
+        sa.ForeignKeyConstraint(['category_id'], ['categories.id'], ),
+        sa.ForeignKeyConstraint(['user_id'], ['users.id'], )
+    )
+    op.create_table(
+        'alert_categories',
+        sa.Column('alert_id', sa.Integer(), nullable=True),
+        sa.Column('category_id', sa.Integer(), nullable=True),
+        sa.ForeignKeyConstraint(['alert_id'], ['alerts.id'], ),
+        sa.ForeignKeyConstraint(['category_id'], ['categories.id'], )
+    )
+    op.alter_column(
+        u'alerts', 'age', existing_type=sa.INTEGER(), nullable=False)
```

Comment:
This seems odd. What's going on with this `alter_column`? I don't think that column was modified.